AutomatedTesting/Gem/PythonTests/terrain/EditorScripts/TerrainPhysicsCollider_ChangesSizeWithAxisAlignedBoxShapeChanges.py

```diff
@@ -0,0 +1,85 @@
+#fmt: off
+class Tests():
+    create_test_entity                      = ("Entity created successfully",    "Failed to create Entity")
+    add_axis_aligned_box_shape              = ("Axis Aligned Box Shape component added", "Failed to add Axis Aligned Box Shape component")
+    add_terrain_collider                    = ("Terrain Physics Heightfield Collider component added", "Failed to add a Terrain Physics Heightfield Collider component")
+    box_dimensions_changed                  = ("Aabb dimensions changed successfully", "Failed change Aabb dimensions")
+    configuration_changed                   = ("Terrain size changed successfully", "Failed terrain size change")
+    no_errors_and_warnings_found            = ("No errors and warnings found",   "Found errors and warnings")
+#fmt: on
+
+def TerrainPhysicsCollider_ChangesSizeWithAxisAlignedBoxShapeChanges():
+    """
+    Summary:
+    Test aspects of the TerrainHeightGradientList through the BehaviorContext and the Property Tree.
+
+    Test Steps:
+    Expected Behavior:
+    The Editor is stable there are no warnings or errors.
+
+    Test Steps:
+     1) Load the base level
+     2) Create test entity
+     3) Start the Tracer to catch any errors and warnings
+     4) Add the Axis Aligned Box Shape and Terrain Physics Heightfield Collider components
+     5) Change the Axis Aligned Box Shape dimensions
+     6) Chaeck the Heightfield provider is returning the correct size
+     7) Verify there are no errors and warnings in the logs
+
+
+    :return: None
+    """
+
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    from editor_python_test_tools.utils import TestHelper as helper
+    from editor_python_test_tools.utils import Report
+    from editor_python_test_tools.utils import Tracer
+    import azlmbr.legacy.general as general
+    import azlmbr.physics as physics
+    import azlmbr.math as azmath
+    import azlmbr.bus as bus
+    import sys
+    import math
+
+    SET_BOX_X_SIZE = 5.0
+    SET_BOX_Y_SIZE = 6.0
+    EXPECTED_COLUMN_SIZE = SET_BOX_X_SIZE + 1
+    EXPECTED_ROW_SIZE = SET_BOX_Y_SIZE + 1
+    helper.init_idle()
+    
+    # 1) Load the level
+    helper.open_level("", "Base")
+
+    # 2) Create test entity
+    test_entity = EditorEntity.create_editor_entity("TestEntity")
+    Report.result(Tests.create_test_entity, test_entity.id.IsValid())
+
+    # 3) Start the Tracer to catch any errors and warnings
+    with Tracer() as section_tracer:
+        # 4) Add the Axis Aligned Box Shape and Terrain Physics Heightfield Collider components
+        aaBoxShape_component = test_entity.add_component("Axis Aligned Box Shape")
+        Report.result(Tests.add_axis_aligned_box_shape, test_entity.has_component("Axis Aligned Box Shape"))
+        terrainPhysics_component = test_entity.add_component("Terrain Physics Heightfield Collider")
+        Report.result(Tests.add_terrain_collider, test_entity.has_component("Terrain Physics Heightfield Collider"))
```

Comment:
nit: makes easier to read:
```suggestion
        aaBoxShape_component = test_entity.add_component("Axis Aligned Box Shape")
        Report.result(Tests.add_axis_aligned_box_shape, test_entity.has_component("Axis Aligned Box Shape"))
        
        terrainPhysics_component = test_entity.add_component("Terrain Physics Heightfield Collider")
        Report.result(Tests.add_terrain_collider, test_entity.has_component("Terrain Physics Heightfield Collider"))
```

Reply:
Done